src/types/annotator.js

```diff
@@ -72,6 +72,36 @@
  * @prop {(ann: AnnotationData) => Promise<Anchor[]>} anchor
  */
 
+/**
+ * @typedef SidebarLayout
+ * @prop {boolean} expanded
```

Comment:
Could you add a description? I guess this is if the sidebar is open or closed, correct?

And the width of the sidebar in pixels correct?

Reply:
Yes, I'll update this.